lib/oat/adapters/json_api.rb

```diff
@@ -80,8 +80,9 @@ def entity(name, obj, serializer_class = nil, context_options = {}, &block)
         end
       end
 
-      def entities(name, collection, serializer_class = nil, context_options = {}, &block)
+      def entities(name, collection, serializer_class = nil, context_options = nil, &block)
         return if collection.nil? || collection.empty?
+        context_options ||= {}
```

Comment:
If the `content_options` argument default value is `nil`, this line will always resolve to `{}`. So keeping the default value as `{}` means this line is not necessary.